backend/src/main/java/net/finman/controller/InvoiceController.java

```diff
@@ -21,4 +22,9 @@
         invoiceService.createInvoice(inv);
         return new ResponseEntity<>(HttpStatus.CREATED);
     }
+
+    @GetMapping("/invoices/{buyer}")
+    public ResponseEntity<?> getInvoices(@PathVariable int buyer) throws ResourceNotCreatedException, ResourceNotFoundException {
+        return new ResponseEntity<>(invoiceService.getInvoices(buyer), HttpStatus.FOUND);
+    }
```

Comment:
is it supposed to throw ResourceNotCreatedException?

Reply:
Welp, that's the flaw of copy-paste programming

Reply:
fixed 👍🏿